src/rust/bitbox02-rust/src/hww/api/restore.rs

```diff
@@ -20,25 +20,42 @@ use pb::response::Response;
 use crate::workflow::{confirm, mnemonic, password, status, unlock};
 
 pub async fn from_file(request: &pb::RestoreBackupRequest) -> Result<Response, Error> {
+    confirm::confirm(&confirm::Params {
+        body: "Restore backup?",
+        accept_is_nextarrow: true,
+        ..Default::default()
+    })
+    .await?;
+
+    let data = match bitbox02::backup::restore_from_directory(&request.id) {
+        Ok(data) => data,
+        Err(_) => {
+            status::status("Could not\nrestore backup", false).await;
+            return Err(Error::Generic);
+        }
+    };
+
+    confirm::confirm(&confirm::Params {
+        body: &format!("Name: {}. ID: {}", &data.name, &request.id),
```

Comment:
i wonder why not combine this with the first one?

```rust
let data = match bitbox02::backup::restore_from_directory(&request.id) { ... }
confirm::confirm(&confirm::Params {
  title: "Restore backup?",
  body: &format!("Name: {}\nID: {}", &data.name, &request.id),
  ...
})
.await?;
```

it just seems like a faster and possibly less confusing UX. less confusing because if i see a blank question "restore backup?" i might assume that's it, there's no more confirmations and it'll start restoring.

Reply:
anyway, wouldn't it be nicer to insert a newline? might as well remove `Name:`. i've just tried, looks better imho:

```rust
body: &format!("{}\nID: {}", &data.name, &request.id),
```